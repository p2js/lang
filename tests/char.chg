fn main(): u8 {
    return b'A' + b'\n';
}
