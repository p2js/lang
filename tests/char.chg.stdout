expr @ 2:12 2:16 => u8
expr @ 2:19 2:24 => u8
expr @ 2:12 2:24 => u8
expr @ 2:5 2:24 => never
expr @ 1:15 4:1 => ()



define i8 @main() {
prologue:
%l.0 = alloca i8
br label %bb.0

bb.0:
%l.1 = add i8 65, 10
store i8 %l.1, i8* %l.0
%ret.0 = load i8, i8* %l.0
ret i8 %ret.0

}


